nextcord/member.py

```diff
@@ -156,7 +157,7 @@ def flatten_user(cls):
             # However I'm not sure how I feel about "functions" returning properties
             # It probably breaks something in Sphinx.
             # probably a member function by now
-            def generate_function(x):
+            def generate_function(x, value=value):
```

Comment:
Why add this arg?

Reply:
I believe this is [`B203`](https://beta.ruff.rs/docs/rules/function-uses-loop-variable/), which could probably just be `noqa`'d if you like, as this is used straight after. Not actually sure as to why `generate_function` exists and why it can't just be inlined alltogether though.

Reply:
Neither, but still yea

Reply:
Hmm, interesting. I see the reasoning with that link, has this been tested? As if it was relying on non B023 behavior there may be change..?